docs/api/collections.rst

```diff
@@ -18,29 +18,33 @@ A collection is a mapping with the following attribute:
     shortcut: ie ``/buckets/default/collections/contacts`` will be
     the current user contacts.
 
-    Internally the user default bucket is assigned to an id that can
+    Internally the user default bucket is assigned to an ID that can
     later be used to share data from a user personnal bucket.
 
-    Collection on a the default bucket are created silently on first access.
+    Collection on a the bucket named "default" are created silently on
```

Comment:
"on a the" is almost certainly a mistake. :smile: If I'm interpreting this phrase properly (and I may not be, heh), consider:
> Collections on the "default" bucket are created silently upon first access and therefore don't need to be set beforehand.